2020/hamming-codes/spanish/sentence_translations.json

```
@@ -506,7 +506,7 @@
  {
   "input": "Keep in mind, though, there is no method for error detection or correction that could give you 100% confidence that the message you receive is the one the sender intended.",
   "model": "nmt",
-  "translatedText": "Sin embargo, tenga en cuenta que no existe ningún método de detección o corrección de errores que pueda brindarle un 100 % de confianza de que el mensaje que recibe es el que pretendía el remitente.",
+  "translatedText": "Sin embargo, hay que tener en cuenta que no existe ningún método de detección o corrección de errores que pueda brindarle un 100 % de confianza de que el mensaje que se recibe es el que pretendía el remitente.",
```

From the original text, I assume "brindarle" also needs to be changed to its "tú" version:

```suggestion
  "translatedText": "Sin embargo, hay que tener en cuenta que no existe ningún método de detección o corrección de errores que pueda brindarte un 100 % de confianza de que el mensaje que se recibe es el que pretendía el remitente.",
```

Perhaps:

```suggestion
  "translatedText": "Sin embargo, ten en cuenta que no existe ningún método de detección o corrección de errores que pueda brindarte un 100 % de confianza de que el mensaje que recibes es el que pretendía el remitente.",
```